apps/web/src/services/workspace-home/components/WorkspaceInfo.vue

```diff
@@ -1,32 +1,122 @@
+
+import type { ListResponse } from '@/schema/_common/api-verbs/list';
+import type { AppListParameters } from '@/schema/identity/app/api-verbs/list';
+import type { AppModel } from '@/schema/identity/app/model';
+import { ROLE_TYPE } from '@/schema/identity/role/constant';
+import type { WorkspaceUserListParameters } from '@/schema/identity/workspace-user/api-verbs/list';
+import type { WorkspaceUserModel } from '@/schema/identity/workspace-user/model';
 import type { WorkspaceModel } from '@/schema/identity/workspace/model';
 import { store } from '@/store';
+import { i18n } from '@/translations';
+
+import { makeAdminRouteName } from '@/router/helpers/route-helper';
 
+import { useAppContextStore } from '@/store/app-context/app-context-store';
 import { useUserWorkspaceStore } from '@/store/app-context/workspace/user-workspace-store';
+import type { UserState } from '@/store/modules/user/type';
 
+import ErrorHandler from '@/common/composables/error/errorHandler';
 import FavoriteButton from '@/common/modules/favorites/favorite-button/FavoriteButton.vue';
 import { FAVORITE_TYPE } from '@/common/modules/favorites/favorite-button/type';
 import WorkspaceLogoIcon from '@/common/modules/navigations/top-bar/modules/top-bar-header/WorkspaceLogoIcon.vue';
 
 import { gray } from '@/styles/colors';
 
+import { APP_DROPDOWN_MODAL_TYPE } from '@/services/iam/constants/app-constant';
+import { USER_MODAL_TYPE } from '@/services/iam/constants/user-constant';
+import { IAM_ROUTE } from '@/services/iam/routes/route-constant';
+import { useAppPageStore } from '@/services/iam/store/app-page-store';
+import { useUserPageStore } from '@/services/iam/store/user-page-store';
+
 const userWorkspaceStore = useUserWorkspaceStore();
 const userWorkspaceGetters = userWorkspaceStore.getters;
+const userPageStore = useUserPageStore();
+const appPageStore = useAppPageStore();
+const appContextStore = useAppContextStore();
+
+const router = useRouter();
 
 const storeState = reactive({
     isDomainAdmin: computed<boolean>(() => store.getters['user/isDomainAdmin']),
+    getCurrentRoleInfo: computed<UserState>(() => store.getters['user/getCurrentRoleInfo']),
     hasAdminOrWorkspaceOwnerRole: computed<boolean>(() => store.getters['user/hasAdminOrWorkspaceOwnerRole']),
     currentWorkspace: computed<WorkspaceModel|undefined>(() => userWorkspaceGetters.currentWorkspace),
     workspaceList: computed<WorkspaceModel[]>(() => userWorkspaceGetters.workspaceList),
 });
 const state = reactive({
+    isWorkspaceOwner: computed<boolean>(() => storeState.getCurrentRoleInfo.roleType === ROLE_TYPE.WORKSPACE_OWNER),
     selectedWorkspace: computed<WorkspaceModel>(() => storeState.workspaceList.find((workspace) => workspace.workspace_id === storeState.currentWorkspace?.workspace_id) || {} as WorkspaceModel),
+    workspaceUserCnt: undefined as number|undefined,
+    appsCnt: undefined as number|undefined,
```

Comment:
nitpick: It's hard to know what 'cnt' means!😅

Reply:
I think so too~